gamification-github-webapp/src/main/webapp/js/githubConnector.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+export default {
+  name: 'github',
+  title: 'Github Profile',
+  description: 'Connect your Github account so your actions from there can be gamified and recognized',
+  icon: 'mdi-github',
+  initialized: true,
+  isSignedIn: true,
+  identifier: '',
+  user: '',
+  rank: 10,
+  openOauthPopup(connector) {
+    const width = 500;
+    const height = 600;
+    const left = window.innerWidth / 2 - width / 2;
+    const top = window.innerHeight / 2 - height / 2;
+    const authUrl = `https://github.com/login/oauth/authorize?client_id=${connector.apiKey}&redirect_uri=${connector.redirectURL}`;
```

Comment:
`redirectURL` must be a constant provided from here depending on used domain and not provided in connector settings.

```suggestion
    const authUrl = `https://github.com/login/oauth/authorize?client_id=${connector.apiKey}&redirect_uri=${redirectURL}`;
```